Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -383,12 +383,17 @@ public function findBlockEnd($type, $searchIndex, $findEnd = true)
      * Find tokens of given kind.
      *
      * @param int|array $possibleKind kind or array of kind
+     * @param int|null  $start        optional offset
```

Comment:
made it `int` only, default `0`, it is like `findSequence`, why would you default something to `null` if that always means you change it to `0`?